void tomo_db0::IoLoop::run_loop<true, false, false, true, true>()
--- PRE

+++ POST

@@ -595,7 +595,7 @@

 0b3d [ 3] nopl (%rax)
 0b40 [ 5] mov $0x1,%esi
 0b45 [ 3] mov %rbx,%rdi
-0b48 [ 5] call <tomo_db0::IoLoop::arm_accept(tomo_db0::UrKind)>
+0b48 [ 5] call <tomo_db0::IoLoop::fliplisten_retry_accept(tomo_db0::UrKind)>
 0b4d [ 5] jmp <+0x32a>
 0b52 [11] data16 cs nopw 0x0(%rax,%rax,1)
 0b5d [ 3] nopl (%rax)
